jujugui/static/gui/src/app/components/user-profile/_user-profile.scss

```diff
@@ -1,7 +1,67 @@
 .user-profile {
+
     &__create-new {
         float: right;
         font-size: 1rem;
+        width: 400px;
+        text-align: right;
+
+        &.collapsed {
+            .deployment-input__field,
+            .button--inline-neutral.second {
+                width: 0;
+                border-width: 0;
+            }
+            .deployment-input__field {
+                padding: 0;
+            }
+            .button--inline-neutral.first {
```

Comment:
I think it would be better to add an additional contextual classes to piggy back this functional styling. 

In case someone changes neutral to base or positive we don't want this to break functionality.